knowhere/knowhere/index/vector_index/helpers/DynamicResultSet.cpp

```diff
@@ -10,11 +10,10 @@
 // or implied. See the License for the specific language governing permissions and limitations under the License.
 
 #include <cstring>
-#include <iostream>
 #include <string>
 #include <utility>
 
-#include "faiss/impl/AuxIndexStructures.h"
```

Comment:
it's wrong to change this header file path.
and this header file should be embraced by "#ifdef __linux__"